client/cypress/e2e/allocationRewardsBox.cy.ts

```diff
@@ -0,0 +1,479 @@
+        .then($el => $el.css('border-color'))
+        .should('be.colored', '#FF6157');
+
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').should('have.value', '100');
+
+      cy.get('[data-test=AllocationInputs__Button]').should('be.disabled');
+
+      cy.get('[data-test=AllocationInputs__InputText--crypto]').clear();
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').should('have.value', '0');
+
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').clear();
+      cy.get('[data-test=AllocationInputs__InputText--crypto]').should('have.value', '0');
+
+      // 50 %
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').type('50');
+      cy.get('[data-test=AllocationInputs__InputText--crypto]').should('have.value', '0.000000005');
+
+      cy.get('[data-test=AllocationInputs__Button]').should('not.be.disabled');
+      cy.get('[data-test=AllocationInputs__Button]').click();
+
+      cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+        .invoke('text')
+        .should('eq', '5 GWEI');
+      cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+        .invoke('text')
+        .should('eq', '5 GWEI');
+
+      cy.get('[data-test=AllocationRewardsBox__section--0]').click();
+
+      cy.get('[data-test=ModalAllocationValuesEdit__header]')
+        .invoke('text')
+        .should('eq', 'Donate 50%');
+
+      // 100 %
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').clear();
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').type('100');
+      cy.get('[data-test=AllocationInputs__InputText--crypto]').should('have.value', '0.00000001');
+
+      cy.get('[data-test=AllocationInputs__Button]').should('not.be.disabled');
+      cy.get('[data-test=AllocationInputs__Button]').click();
+
+      cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+        .invoke('text')
+        .should('eq', '10 GWEI');
+      cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+        .invoke('text')
+        .should('eq', '0 ETH');
+
+      cy.get('[data-test=AllocationRewardsBox__section--0]').click();
+
+      cy.get('[data-test=ModalAllocationValuesEdit__header]')
+        .invoke('text')
+        .should('eq', 'Donate 100%');
+
+      // 1000 %
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').clear();
+      cy.get('[data-test=AllocationInputs__InputText--percentage]').type('1000');
+      cy.get('[data-test=AllocationInputs__InputText--crypto]').should('have.value', '0.00000001');
+
+      cy.get('[data-test=AllocationInputs__Button]').should('not.be.disabled');
+      cy.get('[data-test=AllocationInputs__Button]').click();
+
+      cy.get('[data-test=AllocationRewardsBox__section__value--0]')
+        .invoke('text')
+        .should('eq', '10 GWEI');
+      cy.get('[data-test=AllocationRewardsBox__section__value--1]')
+        .invoke('text')
+        .should('eq', '0 ETH');
```

Comment:
How is that going? Typing 1000% is not possible, validation forbids that, isn't it?

Reply:
This scenario is for testing this situation.